torch/functional.py

```diff
@@ -1232,6 +1233,10 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
         >>> torch.norm(d[0, :, :]), torch.norm(d[1, :, :])
         (tensor(3.7417), tensor(11.2250))
     """
+    warnings.warn((
+        "torch.norm and torch.functional.norm are deprecated. Please "
```

Comment:
I don't think we ever documented torch.functional.norm, so so this note doesn't need to mention it.

"torch.norm is deprecated and may be removed in a future PyTorch release. Use torch.linalg.norm instead."

The warning should also appear in the docstring for `torch.norm` (as a warning).